pkg/apis/kops/zz_generated.deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
Don't know why `make apimachinery` removed those lines.

Reply:
You need to use Go 1.17.1

Reply:
Thanks, fixed now.